Dockerfile

```diff
@@ -1,8 +1,9 @@
-ARG base_version=1.2.0
-ARG registry=quay.io
-
-FROM ${registry}/ncigdc/python38-builder:${base_version} as build
+ARG BASE_VERSION=3.0.1
+ARG REGISTRY=quay.io
```

Comment:
I might be missing something but I saw other repos are using:
`REGISTRY=docker.osdc.io`

Reply:
that is used after travis is removed.